### DISCUSSION TITLE
бага inputdaterange в мобилке

### DISCUSSION THREAD

**Discussion by said4505** [taiga-family/taiga-ui#5287] [General]
Использую в проекте tui-input-date-range. В мобилке отключил mobilecalendarmodule так как фуллскрин модалка не нужна, вот это окошко с календарем появляется только если очистить дату, в десктопе открывается при фокусе, как решить эту проблему? 
![image](https://github.com/taiga-family/taiga-ui/assets/76432281/59ec8c60-726c-4561-939c-5ebf82aedf56)

**Comment by waterplea** [comment]
You can provide `false` with some directive:
```ts
{
  provide: TUI_IS_MOBILE,
  useValue: false,
}
```
This way component will not know it is on a mobile device.
